media/css/mozorg/home/home-new.scss

```diff
@@ -14,6 +14,14 @@ header {
 
         h1 {
             @include font-size(54px);
+
+            @media #{$mq-xs} {
+               @include font-size(38px);
```

Comment:
This could be the default size with no media query, then it just enlarges at the wider breakpoint. That's a more mobile-first approach where we assume small screens by default and let things get bigger when we know there's room (and also caters to less capable browsers that may not support media queries).

We should probably also adjust the size of the paragraph below this title on smaller screens.